bootloader/grub_test.go

```diff
@@ -605,3 +606,131 @@ managed by snapd`))
 	c.Assert(err, ErrorMatches, "cannot load existing config asset: .*/grub.cfg: permission denied")
 	c.Assert(is, Equals, false)
 }
+
+func (s *grubTestSuite) TestRecoveryUpdateBootConfigNoEdition(c *C) {
```

Comment:
shouldn't we have a positive case also for the Recovery case?

Reply:
Added